### PR TITLE
Add NoteFlight

### PR DESCRIPTION
**Pull request by Anshuman-Verma**
**https://www.noteflight.com/login**

**Print music sheets, write your own music online (review).**

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- [x] I have read the [Contribution guidelines](https://github.com/aviaryan/awesome-no-login-web-apps/blob/master/CONTRIBUTING.md) and I am confident that my PR is suitable. 
- [x] This pull request has a descriptive title. For example, `Add [App name]`, not `Update readme.md` or `Added new entries`.
- [x] The app I added is not a duplicate.

### CODE REVIEW

**Comment by Anshuman-Verma** [comment]
Doubtful on the section for this web-app.

**Comment by Anshuman-Verma** [comment]
^conflicts resolved

**Comment by aviaryan** [comment]
Maybe it can go in the Music section.

**Comment by Anshuman-Verma** [comment]
^Fixed

File: README.md [hunk 1]
```diff
@@ -125,6 +125,7 @@ Powershell, Redis and lots others. Also has a good collection of IDEs for progra
 * [plug.dj](https://plug.dj/) - Listen to thousands of DJs, powered by YouTube, sign-in required to become a DJ.
 * [Jango Radio](https://jango.com/) - Free Internet radio, listen to hundreds of genre stations or create your own with your favorite music.
 * [MusicPleer](http://musicpleer.cc/) - Stream or download your favorite songs without any hassle.
+* [NoteFlight](https://www.noteflight.com/login) `[Account]` - print music sheets, write your own music online (review).
```
Comment (aviaryan) [inline README.md]: Make description start with a capital letter.

**Comment by Anshuman-Verma** [comment]
Please review again

**Comment by Anshuman-Verma** [comment]
Any problem with PR still ?

**Pull request merged**